os/android/utils_android.h

```diff
@@ -251,6 +251,11 @@ static bool ImportGraphicsBuffer(HWCNativeHandle handle, int fd) {
   if (handle->meta_data_.format_ == DRM_FORMAT_YVU420_ANDROID)
     handle->meta_data_.format_ = DRM_FORMAT_YVU420;
 
+#ifdef USE_GRALLOC1
```

Comment:
Why we need this USE_GRALLOC1 macro?

Reply:
Agree this macro is not needed since HwcBuffer services both gralloc0 and gralloc1.